test/failover/failover.test.lua

```diff
@@ -177,6 +177,7 @@ cfg.failover_ping_timeout = 0.0000001
 vshard.router.cfg(cfg)
 while not test_run:grep_log('router_1', 'Ping error from', 1000) do fiber.sleep(0.01) end
 
+collectgarbage("stop")
```

Comment:
What do you mean as an error in this test? AFAIR, netbox won't delete itself while at least one request is in progress. Which means the "long" request should complete fine even if the connection has no refs outside of itself. No?

Reply:
This is how it works, I don't know the reason, why the connection is closed by Lua GC. This is the bug, which should be fixed, created ticket for that: https://github.com/tarantool/tarantool/issues/9629